.github/workflows/CI.yml

```diff
@@ -44,6 +44,23 @@ jobs:
       - uses: gap-actions/process-coverage@v2
       - uses: codecov/codecov-action@v2
 
+  test-cygwin:
+    name: "master cygwin64 HPC no"
+    if: ${{ !(github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository) }}
+    runs-on: windows-latest
+    env:
+      CHERE_INVOKING: 1
```

Comment:
Just to understand: why do we need this here? don't all the actions already set it?

Reply:
This is probably not necessary here. However, in my opinion it is not worth my time to remove it and wait to see if it works.

However, I think it _would_ be required if one ever added a step to this job that ran some commands directly in the Cygwin bash shell. Keeping this (otherwise harmless) environment variable set means that adding such a step would be as seamless as possible.

Reply:
It seems it does need to be there, I don't really understand why, but if it's not then `gap-actions/build-pkg` and `gap-actions/run-pkg-tests` start in the GAP directory, and not in the ace directory, and so they fail at building the package and running its tests, respectively.